src/libP/solvers/elliptic/okl/ellipticSerialAxHex3D.c

```diff
@@ -118,6 +118,100 @@ void ellipticAxHex3D(const dlong & Nelements,
         }
   }
 }
+extern "C"
+void ellipticAxFloatHex3D(const dlong & Nelements,
```

Comment:
I don't think we need an extra float version. Just compile the kernel with a different dfloat type.